tests/fixtures/saas/aircall_fixtures.py

```diff
@@ -0,0 +1,152 @@
+from time import sleep
+from typing import Any, Dict, Generator
+
+import pydash
+import pytest
+import requests
+from sqlalchemy.orm import Session
+
+from fides.api.ctl.sql_models import Dataset as CtlDataset
+from fides.api.ops.models.connectionconfig import (
+    AccessLevel,
+    ConnectionConfig,
+    ConnectionType,
+)
+from fides.api.ops.models.datasetconfig import DatasetConfig
+from fides.api.ops.util.saas_util import (
+    load_config_with_replacement,
+    load_dataset_with_replacement,
+)
+from fides.lib.cryptography import cryptographic_util
+from tests.ops.test_helpers.vault_client import get_secrets
+
+secrets = get_secrets("aircall")
+
+
+@pytest.fixture(scope="session")
+def aircall_secrets(saas_config):
+    return {
+        "domain": pydash.get(saas_config, "aircall.domain") or secrets["domain"],        
+        "api_token": pydash.get(saas_config, "aircall.api_token") or secrets["api_token"],
```

Comment:
Update this with `app_id` and `api_token`

Reply:
updated app_id and api_token in aircall_fixtures.py